ga4gh/protocol.py

```diff
@@ -5,14 +5,46 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
-import sys
+import datetime
 import json
 import inspect
-import datetime
-import itertools
-from cStringIO import StringIO
 
-import avro.io
+import google.protobuf.json_format as json_format
```

Comment:
Use ``import google.protobuf as protobuf`` here, and use `protobuf.json_format()`` in the code. This style choice is good because it means we can read the code using the function and locally see where the function in question comes from. Likewise for ``message``.